.github/workflows/update-dependencies.yml

```diff
@@ -52,14 +42,14 @@ jobs:
   notify-on-failure:
     name: Slack notify on failure
     if: failure()
-    needs: [ update-dependencies-and-test, commit-dependencies ]
+    needs: [ update-dependencies-and-test ]
     runs-on: ubuntu-latest
     steps:
       - uses: rtCamp/action-slack-notify@v2
         env:
           SLACK_COLOR: failure
           SLACK_ICON_EMOJI: ':github:'
-          SLACK_MESSAGE: ${{ format('Update dependencies *{0}*, commit dependencies *{1}*, {2}/{3}/actions/runs/{4}', needs.update-dependencies-and-test.result, needs.commit-dependencies.result, github.server_url, github.repository, github.run_id) }}
+          SLACK_MESSAGE: ${{ format('Update dependencies *{0}*, {2}/{3}/actions/runs/{4}', needs.update-dependencies-and-test.result, github.server_url, github.repository, github.run_id) }}
```

Comment:
Indices here need be updated. E.g., `{1}` is missing which is now `github.server_url`